### DISCUSSION TITLE
Generation of C++ classes from C#

### DISCUSSION THREAD

**Discussion by CeSun** [nxrighthere/UnrealCLR#250] [Help] [answered]
Can I use c# like c++ by unrealclr plugin?

**Comment by nxrighthere** [comment, marked as answer]
It's not supported at the moment, please, see https://github.com/nxrighthere/UnrealCLR/discussions/171.

**Reply by CeSun** [reply]
Think you.

**Comment by nicholasneborovsky** [comment]
Can I use C++ on my Visual Studio Code project?

**Reply by CeSun** [reply]
I don't understand what you mean.

**Reply by nxrighthere** [reply]
@nicholasneborovsky How do you mean?
